### DISCUSSION TITLE
dating a tree with cross bracing

### DISCUSSION THREAD

**Discussion by bonisko** [iqtree/iqtree2#324] [General]
Hi all!

I have been reading papers where a calibration date is duplicated in a tree, and then the dating uses "cross-bracing" to require both nodes have the same date.  (See for example: https://www.pnas.org/doi/full/10.1073/pnas.1305813110)

Does anyone know if I can do this in IqTree with LSD dating?

If not, would implementation be possible?

Thanks!

Bruce

**Comment by bonisko** [comment]
just tagging @tothuhien

**Comment by roblanf** [comment]
@bonisko I'm 99% sure you can't do this in LSD2. You'll probably need to use the cross-bracing software from the bacteria paper(s) where it's presented.

**Comment by tothuhien** [comment]
Hi @bonisko , sorry forgot about this one and thanks @roblanf for the answer. It's true that lsd2 does not have this option. It maybe something feasible to include in the model but the implementation would take time and I'm really not sure when I can.
Hien
